### PR TITLE
setup profile for video/picture items

### PR DESCRIPTION
**Pull request by petrjasek**
and populate default values using client extension

SDBELGA-280

### CODE REVIEW

File: client/extensions/iptc/src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const DEFAULT_SUBJECT = {
+    'media-credit': ['BELGA_ON_THE_SPOT'],
+    'belga-keywords': ['RUSHES', 'BELGAILLUSTRATION', 'BELGAINTERVIEW', 'BELGAINSERT'],
+    'countries': ['BEL'],
+};
+
+const extension: IExtension = {
+    id: 'iptc',
+    activate: (superdesk: ISuperdesk) => {
+        console.info('activate');
```
Comment (ride90) [inline client/extensions/iptc/src/extension.ts]: forgotten console

File: client/extensions/iptc/src/extension.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import {IExtension, IExtensionActivationResult, ISuperdesk, IArticle, ISubject} from 'superdesk-api';
+
+type Scheme = 'media-credit' | 'belga-keywords' | 'countries';
+
+const CVS: Array<Scheme> = ['media-credit', 'belga-keywords', 'countries'];
+
+const DEFAULT_SUBJECT = {
+    'media-credit': ['BELGA_ON_THE_SPOT'],
+    'belga-keywords': ['RUSHES', 'BELGAILLUSTRATION', 'BELGAINTERVIEW', 'BELGAINSERT'],
+    'countries': ['BEL'],
```
Comment (ride90) [inline client/extensions/iptc/src/extension.ts]: If you need a single selection, you can use a `country` CV https://github.com/superdesk/superdesk-belga/blob/master/server/data/vocabularies.json#L4204

File: server/picture-profile.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            "order": 7,
+            "section": "header"
+        },
+        "event_description": {
+            "order": 8,
+            "section": "header",
+            "sdWidth": "full",
+            "textarea": true
+        },
+        "source": {
```
Comment (ride90) [inline server/picture-profile.json]: if `source` is always "Belga" and filled with extension anyway, I would remove it from the editor.

File: server/data/vocabularies.json [hunk 4]
```diff
@@ -11874,6 +11874,25 @@
             }
         ]
     },
+    {
+        "_id": "media-credit",
```
Comment (ride90) [inline server/data/vocabularies.json]: I think 
```
"service": { "all": 1}
```
is missed.

File: server/picture-profile.json (hunk 3 above)
Comment (petrjasek) [reply server/picture-profile.json]: I think if it's not there it won't make it to the formatter

Comment (ride90) [reply server/picture-profile.json]: I've tried and it worked. I hope it was not cache :)

Comment (petrjasek) [reply server/picture-profile.json]: ok thx for checking

**Pull request merged**